Source/Core/VideoCommon/PixelShaderManager.cpp

```diff
@@ -447,17 +447,17 @@ void PixelShaderManager::SetGenModeChanged()
   dirty = true;
 }
 
-void PixelShaderManager::SetZControlChanged()
+void PixelShaderManager::SetZModeControl()
```

Comment:
I forgot to add the call to it when the corresponding register was updated in the initial PR.